Riot/Modules/Rooms/RoomsViewController.m

```diff
@@ -106,6 +106,19 @@ - (void)onPlusButtonPressed
     self.currentAlertController = [self showPlusMenuFrom:self->plusButtonImageView];
 }
 
+// Tchap: Fix dataSource for pagination.
+- (void)displayList:(MXKRecentsDataSource *)listDataSource {
+    [super displayList:listDataSource];
+    
+    if ([self.dataSource isKindOfClass:RecentsDataSource.class])
+    {
+        // Take the lead on the shared data source.
+        recentsDataSource = (RecentsDataSource*)self.dataSource;
+        recentsDataSource.areSectionsShrinkable = NO;
```

Comment:
Why don't we use shrinkable sections?

Reply:
would be done later (https://github.com/tchapgouv/tchap-ios/issues/495)

Reply:
Because there are few bugs in this implementation (from Element 1.8.3), and they are fixed in newer Element versions (starting 1.8.4).
So we had to rebase onto a newer Element version before enabling them on Tchap.